pages/linux/sa.md

```diff
@@ -0,0 +1,16 @@
+# sa
+
+> Summarizes accounting information. Part of the acct package.
+> Shows commands called by users, including basic info on CPU time spent processing and I/O rates.
+
+- Display executable invocations by every user:
+
+`sudo sa`
+
+- Display executable invocations by every user with username:
```

Comment:
Specifically what does this command do differently to the previous one? I'm not sure I understand the description here.

Reply:
Invoking `sa` does not resolve user-ids into readable usernames.

You think it is preferable to modify the text in the very first example - the lack of usernames might surprise users coming to the tool for the first time. In the name of a shorter learning curve a slightly higher bump at the start might be appropriate?

Reply:
That sounds like a good plan, yeah. Let's do that.